ansible/playbooks/tasks/iscsi-server-sbd-prep.yaml

```diff
@@ -9,27 +9,30 @@
       - python-configshell
       - targetcli
     
-  - name: Install required packages on iscsi server SLES 15.3
+  - name: Install iscsi server packages SLES 15.3
     community.general.zypper:
       name: "{{ item }}"
       state: present
-    with_items:
+    loop:
```

Comment:
There are a bunch of technical reasons, but the main reason is the loops are newer and ansible are encouraging the use of loop rather than with_*.  Nothing has been deprecated yet, but the older with_* style loops may be deprecated in the future.  In the majority of our cases, using loop will help to future-proof the code with no negative side effects.

https://docs.ansible.com/ansible/latest/user_guide/playbooks_loops.html